src/SprykerSdk/Zed/AppSdk/Business/Manifest/Builder/ManifestBuilder.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Business\Manifest\Builder;
+
+use Generated\Shared\Transfer\ManifestRequestTransfer;
+use Generated\Shared\Transfer\ManifestResponseTransfer;
+
+class ManifestBuilder implements ManifestBuilderInterface
+{
+    /**
+     * @var array<string>
+     */
+    protected $transferToManifestTypeMap = [
```

Comment:
Why is this here? You don't need this code and when you run your tests with `--coverage-html` and look at the created coverage in your browser, you will see that this line is not used.